test/Command/GenerateProgrammaticPipelineFromConfig/GeneratorTest.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2015-2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendTest\Expressive\Command\GenerateProgrammaticPipelineFromConfig;
+
+use org\bovigo\vfs\vfsStream;
+use org\bovigo\vfs\vfsStreamDirectory;
```

Comment:
Odd, I needed to have this in another test suite whenever I used the `vfsStream::setup()` method; evidently that's no longer true in recent versions. Removing.